netbout-client/src/main/java/com/netbout/client/RtAttachment.java

```diff
@@ -116,6 +117,24 @@ public boolean unseen() throws IOException {
         );
     }
 
+    // There is a puzzle about this method in
+    // DyAttachmentITCase.obtainsCreationDate()
```

Comment:
@CarlosGines we generally shouldn't leave such comments, as they may be forgotten and not removed with a puzzle